typings/contentFields.d.ts

```diff
@@ -1,8 +1,49 @@
-export interface ContentFields {
-  id: string,
-  name: string,
-  required: boolean,
-  localized: boolean,
-  type: string,
+export interface NumRange {
+  min: number
```

Comment:
I think we should allow null and not set option here `?`
because If I remember correctly you can set only min or max